readme.md

```diff
@@ -127,11 +127,37 @@ Run `brew cask install provisionql` or [download manually](https://github.com/ea
 
 [![](screenshots/ProvisionQL.png)](https://github.com/ealeksandrov/ProvisionQL)
 
+
 ### Other
 
 - [CertQuickLook](https://code.google.com/p/cert-quicklook/) - preview various unprotected certificate tokens like X509 certificates, DER or PEM
 
 
+## Tip
+
+> After running any of the following commands you will need to relaunch Finder for the changes to take affect.  You can do so quickly with `killall Finder`.
+
+### Enable cursor text selection inside Quick Look window
+
+&#8804; 10.10 Yosemite
+
+    defaults write com.apple.finder QLEnableTextSelection -bool TRUE
+
+&#8805; 10.11 El Capitan
+
+    defaults write com.apple.finder QLEnableTextSelection -bool YES
```

Comment:
`-bool TRUE` and `-bool YES` are equivalent and not likely to be the cause of the difference in behavior you’re seeing.